client/state/notices/reducer.js

```diff
@@ -9,28 +9,29 @@ import { combineReducers } from 'redux';
 import {
 	NEW_NOTICE,
 	REMOVE_NOTICE,
-	SET_ROUTE
+	SET_ROUTE,
+	SERIALIZE,
+	DESERIALIZE
 } from 'state/action-types';
 
 export function items( state = [], action ) {
 	switch ( action.type ) {
 		case NEW_NOTICE:
-			state = [ action.notice, ...state ];
-			break;
+			return [ action.notice, ...state ];
```

Comment:
@dmsnell, @bob added https://github.com/Automattic/wp-calypso/pull/2998. We should be able to use returns in our redux switches.